Resources/config/services.xml

```diff
@@ -64,6 +64,8 @@
 
         <parameter key="jms_serializer.accessor_strategy.default.class">JMS\Serializer\Accessor\DefaultAccessorStrategy</parameter>
         <parameter key="jms_serializer.accessor_strategy.expression.class">JMS\Serializer\Accessor\ExpressionAccessorStrategy</parameter>
+
+        <parameter key="jms_serializer.cache.cache_warmer.class">JMS\SerializerBundle\Cache\CacheWarmer</parameter>
```

Comment:
Don't Symfony's best practices say to not use `.class` parameters? Framework extra bundle [removed](https://github.com/sensiolabs/SensioFrameworkExtraBundle/blob/master/CHANGELOG.md#40) them

Reply:
hmm. true

Reply:
Decided to leave this as currently the bundle is still using the old convention. Do not want to mix conventions.

While changing it will be a BC break. Maybe for the version 3.0